qcodes/instrument_drivers/stanford_research/SR86x.py

```diff
@@ -37,7 +37,6 @@ def __init__(self, name: str, instrument: 'SR86x') ->None:
                          docstring='Holds an acquired (part of the) data '
                                    'buffer of one channel.')
 
-        self.name = name
```

Comment:
We should probably check qcodes_contrib_drivers for this pattern before merging

Reply:
Yes, I did not think of checking that. Thanks for the input.

Reply:
I checked qcodes_contrib_driver and did not find any driver assigning new name to `self.name` .